ofono/src/slot-manager.c

```diff
@@ -1340,6 +1340,20 @@ void ofono_slot_drop_cell_info_requests(struct ofono_slot *s, void* tag)
 	}
 }
 
+struct ofono_slot *ofono_slot_find_imsi(const char *imsi)
```

Comment:
I would call this `ofono_slot_get_by_imsi()`. (find is fine too, it's just an implementation detail)

Reply:
`ofono_slot_find_by_imsi()` it is then ;)